packages/browsh.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Browsh < Package
+  description 'A fully-modern text-based browser, rendering to TTY and browsers.'
+  homepage 'https://www.brow.sh'
+  @_ver = '1.6.4'
+  version @_ver
+  license 'LGPL-2.1'
+  compatibility 'all'
+
+  depends_on 'firefox'
+    
+  case ARCH
+  when 'armv7l', 'aarch64'
+    @_arch = 'armv7'
+  when 'x86_64'
+    @_arch = 'amd64'
+  when 'i686'
+    @_arch = '386'
+  end
+    
+  @_file = "browsh_#{@_ver}_linux_#{@_arch}"
+  @_url = "https://github.com/browsh-org/browsh/releases/download/v#{@_ver}/"
+
+  source_url @_url + @_file
+  
+  def self.preflight
+    source_sha256 `curl -Ls #{@_url}/browsh_#{@_ver}_checksums.txt`.scan(/(.*)#{@_file}$/)[0][0].strip
```

Comment:
Do we need to put this in preflight?

Reply:
I feel like this could just go in the normal source_sha256 spot.

Reply:
It is for speeding up actions which doesn't need to download (search, update)

Reply:
I'm not following.  Shouldn't those operations skip it anyway?

Reply:
Although they will skip the download part (and checksum), the content inside the backtick will still execute by ruby

Reply:
Why not just make it static?

Reply:
@supechicken: I think we better make all checksums static going forward.  I see what you mean when the checksum is executed during `crew search`.  This happens on the require line in `crew` as shown below:
```
def set_package (pkgName, pkgPath)
  require pkgPath
```

Reply:
Change made